src/components/resources-section/resource-sidebar/ResourceSidebar.js

```diff
@@ -3,29 +3,30 @@ import { Card } from 'primereact/card';
 import LinkButton from '@components/resource-section/link-button/LinkButton';
 import AddEvaluation from '@components/resource-section/add-evaluation/AddEvaluation';
 import Average from '@components/resources-section/average/Average';
-import styles from './ResourceSidebar.module.scss';
+import EvaluationList from '@components/resource-section/evaluation-list/EvaluationList';
 
 const ResourceSidebar = ({
-  visible,
   onHideHandler,
   activeResource,
   formOptions,
+  evaluations,
 }) => {
   return (
     <Sidebar
-      visible={visible}
+      visible={true}
       position="right"
       onHide={() => onHideHandler()}
-      className={styles.resourceSidebar}
       modal={false}
       dismissable={false}
+      style={{ width: '25%' }}
```

Comment:
Inline style but I understand that it was the only way.

Reply:
thx